packages/wdio-types/src/Options.ts

```diff
@@ -43,24 +43,31 @@ export interface Connection {
     /**
      * Protocol to use when communicating with the Selenium standalone server (or driver).
      *
+     * If you specify a value other than the default, the webdriver won't be started automatically.
```

Comment:
Reference:

https://github.com/webdriverio/webdriverio/blob/aa35c291fb496b80350766d674385ea7b4d1516a/packages/wdio-utils/src/driver/utils.ts#L257-L265

Applies for all other documentation updates in this file.